src/corehost/cli/dll/CMakeLists.txt

```diff
@@ -67,3 +67,4 @@ if (WIN32 AND CLI_CMAKE_PLATFORM_ARCH_ARM)
     target_link_libraries(hostpolicy shell32.lib)
 endif()
 
+install_library_and_symbols (hostpolicy)
```

Comment:
Oh, sorry, the red "no newline at end of file" marker only shows up in the files diff view, so you probably didn't see it. I was just suggesting adding a newline after this line to make linux happy. The empty line before this line looks good to me.